docs/source/schema/unions-interfaces.md

```diff
@@ -1,48 +1,84 @@
 ---
 title: Unions and interfaces
-description: How to add unions and interfaces to a schema
+description: Polymorphic schema types
 ---
 
-Unions and interfaces are great when you have fields that are in common between two types.
+**Unions** and **interfaces** are polymorphic GraphQL types that enable a schema field to contain one of multiple object types.
+
+> **Tip:** If a field can contain object types that **all share relevant subfields**, use an interface. Otherwise, use a union.
 
 ## Union type
 
-The `Union` type indicates that a field can return more than one object type, but doesn't define specific fields itself.  Unions are useful for returning disjoint data types from a single field. The type definitions appear as follows:
+A union type indicates that a field's value might be one of multiple distinct object types. Those object types are _not_ required to share any fields (but they can).
```

Comment:
I don't know that it's helpful to talk about the parent field (here and previously). A union type (and interface type) can be talked about independently of the field that returns it.

If we _do_ want to reference the field, I would call this the field's _return type_ rather than its value.

Here's a rephrase that I don't dislike:
>A field which returns a union type indicates that its value can be one of multiple distinct object types.